index.html

```diff
@@ -2,61 +2,116 @@
+
+        //   var numberOne = parseInt(document.forms.inputs.numberOne.value);
+        //   var numberTwo = parseInt(document.forms.inputs.numberTwo.value);
+        //   document.forms.inputs.result.value = numberOne - numberTwo;
+
+        //     },
+        // }
+
+        let calculator = {
             addNumbers: function () {
-                // hint: these are strings
-                var numberOne = document.forms.inputs.numberOne.value;
-                var numberTwo = document.forms.inputs.numberTwo.value;
+                let numberOne = parseInt(document.forms.inputs.numberOne.value);
+                let numberTwo = parseInt(document.forms.inputs.numberTwo.value);
                 document.forms.inputs.result.value = numberOne + numberTwo;
             },
+
             subtractNumbers: function () {
-                
+                let numberOne = parseInt(document.forms.inputs.numberOne.value);
+                let numberTwo = parseInt(document.forms.inputs.numberTwo.value);
+                document.forms.inputs.result.value = numberOne - numberTwo;
+            },
+
+            multiplyNumbers: function () {
+                let numberOne = parseInt(document.forms.inputs.numberOne.value);
+                let numberTwo = parseInt(document.forms.inputs.numberTwo.value);
+                document.forms.inputs.result.value = numberOne * numberTwo;
+            },
+
+            divideNumbers: function () {
+                let numberOne = parseInt(document.forms.inputs.numberOne.value);
+                let numberTwo = parseInt(document.forms.inputs.numberTwo.value);
+                document.forms.inputs.result.value = numberOne / numberTwo;
             }
         }
 
+
+
     </script>
 
 </head>
 
 <body>
-    <div>Header</div>
+    <H1>Calculator 1.2</H1>
     <div>
-       <!--
+        <!--
          - allow two numbers to be entered
          - show buttons for add and subtract
          - show the result
         -->
         <form name="inputs">
             <div>
-                Number 1:
-                <input type="number" name="numberOne" class="form-control" placeholder="First number" aria-label="Number1" >
+                Enter Number 1:
+                <input type="number" name="numberOne" class="form-control" placeholder="First Number"
+                    aria-label="Number1">
             </div>
             <div>
-                Number 2:
-                <input type="number" class="form-control" name="numberTwo">
+                Enter Number 2:
+                <input type="number" name="numberTwo" class="form-control" placeholder="Second Number"
+                    aria-label="Number2">
             </div>
             <div>
-                <button type="button" class="btn btn-primary" onclick="calculator.addNumbers()" >Add</button>
-                <button type="button" class="btn btn-primary">Substract</button>
+                <button type="button" class="btn btn-primary" padding: 50px 25px
```

Comment:
This all looks good. There is an issue though. The "padding" and related values all has to e removed.  You can add that to the btn, btn-primary or other css class if you want to include padding.  It is not valid to put a direct padding attribute on any html tag.